### DISCUSSION TITLE
✅ 🚀 Feature Request: Add an active class to open Accordion items

### DISCUSSION THREAD

**Discussion by iamchetanp** [elementor/elementor#15994] [Feature Request]
Hi,

I want to add active class on accordion item. Currently, it adds only to Title and Content. It will be good if this feature included.

Thanks,

Ref. https://forum.elementor.com/fix-common-issues-and-report-bugs-33/active-class-on-accordion-item-3408

**Comment by nicholaszein** [comment]
Hello there @iamchetanp!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new nested Accordion in Elementor v3.14! 🥳 

You can now use the following CSS to target the content inside opened nested Accordions:

```css
/* Handles the nested Accordion item Title when opened */
.e-n-accordion[open] > .e-n-accordion-item-title {
    /* do something */
}

/* Handles the nested Accordion item content wrapper Container when opened */
.e-n-accordion[open] > .e-child {
    /* do something */
}
```

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
